build/findbugs/findbugs-exclude.xml

```diff
@@ -2,13 +2,7 @@
 
   <Match>
     <Or>
-      <Package name="org.apache.thrift"/>
```

Comment:
we don't actually have this package now, right?

Reply:
We don't